app/assets/stylesheets/modules/applicationfooter.scss

```diff
@@ -0,0 +1,32 @@
+@import "variables";
+
+.application-footer {
+  background: $blue;
+  bottom: 0;
+  left: 0;
```

Comment:
Properties should be ordered background, bottom, color, height, left, line-height, position, text-align, width